.ci/Jenkinsfile

```diff
@@ -227,6 +230,7 @@ pipeline {
   }
   post {
     cleanup {
+      githubCheckNotify(currentBuild.currentResult == 'SUCCESS' ? 'SUCCESS' : 'FAILURE')
```

Comment:
I'd say to add the 'PENDING' check in [L77](https://github.com/elastic/e2e-testing/pull/315/files#diff-8db37e8fcea0f1a8f2f39667e94ebcc4L77)

Reply:
It's set here in the upstream job: https://github.com/elastic/beats/pull/21100/files#diff-bdaa5a61de14efda6eb2bbb2a38fa9a8R269

Should you suggest adding it here and remove it from there? I like the idea of having the control in one single place

Reply:
I'm trying to recall if there is any scenario that will be required, but nothing on the top off my head, let's then leave it as is

Reply:
In the other hand, I think it's harmless to add it here instead of in the packaging job: for PRs in this repo, it will no trigger this check; for upstream PRs, it will trigger the check

Reply:
I've checked that the opbeans pipeline follows the same approach (https://github.com/elastic/apm-pipeline-library/blob/f0b506f783e0fe5b468d034d6459937fc981f806/vars/opbeansPipeline.groovy#L202-L208):
- trigger the downstream job
- start the pending check
which is the same as we do for this use case: the packaging job triggers this job and starts the pending check.